backend/layers/processing/process_validate_atac.py

```diff
@@ -0,0 +1,224 @@
+            key = key_prefix + "." + ARTIFACT_TO_EXTENSION[artifact_type]
+            datasets_s3_uri = self.upload_artifact(file_name, key, datasets_bucket)
+            self.logger.info(f"Uploaded {dataset_version_id}.{artifact_type} to {datasets_s3_uri}")
+            self.business_logic.update_dataset_artifact(artifact_id, datasets_s3_uri)
+            self.logger.info(f"Updated database with {artifact_type}.")
+            self.update_processing_status(dataset_version_id, processing_status_key, DatasetConversionStatus.UPLOADED)
+            return artifact_id
+        except Exception as e:
+            self.logger.error(e)
+            raise ConversionFailed(processing_status_key) from None
+
+    def skip_atac_validation(
+        self, local_anndata_filename: str, manifest: IngestionManifest, dataset_version_id
+    ) -> bool:
+        """
+        Check if atac validation should be skipped
+        :param local_anndata_filename: the local anndata file
+        :param manifest: the manifest
+        :param dataset_version_id: the dataset version id
+        :return: True if the validation should be skipped, False otherwise
+        """
+        # check if the anndata should have a fragment file
+        try:
+            result = self.schema_validator.check_anndata_requires_fragment(local_anndata_filename)
+        except ValueError as e:  # fragment file forbidden
+            self.logger.warning(f"Anndata does not support atac fragment files for the follow reason: {e}")
+            if manifest.atac_fragment:
+                self.update_processing_status(
+                    dataset_version_id,
+                    DatasetStatusKey.VALIDATION,
+                    DatasetValidationStatus.INVALID,
+                )
+                raise ValidationAtacFailed(errors=[str(e), "Fragment file not allowed for non atac anndata."]) from None
+            self.logger.warning("Skipping fragment validation")
+            self.update_processing_status(
+                dataset_version_id,
+                DatasetStatusKey.ATAC_FRAGMENT,
+                DatasetConversionStatus.SKIPPED,
+                validation_errors=[str(e)],
+            )
+            return True
+
+        if manifest.atac_fragment is None:
+            if result:  # fragment file required
+                self.update_processing_status(
+                    dataset_version_id, DatasetStatusKey.VALIDATION, DatasetValidationStatus.INVALID
+                )
+                raise ValidationAtacFailed(errors=["Anndata requires fragment file"])
+            else:  # fragment file optional
+                self.logger.info("Fragment is optional and not present. Skipping fragment validation.")
+                self.update_processing_status(
+                    dataset_version_id,
+                    DatasetStatusKey.ATAC_FRAGMENT,
+                    DatasetConversionStatus.SKIPPED,
+                    validation_errors=["Fragment is optional and not present."],
```

Comment:
conversely, this seems to me like the correct usage of SKIPPED. atac IS applicable, but was skipped since it was optional